src/utils/update/managing.officer.mapper.ts

```diff
@@ -39,7 +39,7 @@ export const mapToManagingOfficerCorporate = (officer: CompanyOfficer): Managing
     ch_reference: mapSelfLink(officer.links?.self),
     name: officer.name,
     principal_address: address,
-    is_service_address_same_as_principal_address: isSameAddress(service_address, address) ? yesNoResponse.Yes : yesNoResponse.No,
+    is_service_address_same_as_principal_address: undefined,
```

Comment:
if we wanted to future proof it we could do:
 isSameAddress(service_address, address) ? yesNoResponse.Yes : undefined
But ok for now.